sodiumoxide/sodiumoxide.rs

```diff
@@ -0,0 +1,58 @@
+#![feature(test)]
+
+extern crate test;
+
+#[macro_use]
+extern crate crypto_bench;
+
+extern crate sodiumoxide;
+
+mod digest {
+    macro_rules! sodiumoxide_digest_benches {
+        ( $name:ident, $block_len:expr, $output_len:expr, $digester:path) => {
+            mod $name {
+                use crypto_bench;
+                use $digester;
+
+                digest_benches!($block_len, input, {
+                    hash(&input)
+                });
+            }
+        }
+    }
+
+    sodiumoxide_digest_benches!(sha256, crypto_bench::SHA256_BLOCK_LEN,
+                         crypto_bench::SHA256_OUTPUT_LEN,
+                         sodiumoxide::crypto::hash::sha256::hash);
```

Comment:
Could we fix the alignment here? If the names are too long, then let's do it like this:

```
    sodiumoxide_digest_benches!(
        sha256, crypto_bench::SHA256_BLOCK_LEN,
        crypto_bench::SHA256_OUTPUT_LEN,
        sodiumoxide::crypto::hash::sha256::hash);
```